### PR TITLE
Adding usage analysis with application insights

### PR DESCRIPTION
**Pull request by jonocairns**

### CODE REVIEW

File: docs/HOL-Usage_Analysis_With_Application_Insights/README.md [hunk 1]
```diff
@@ -0,0 +1,377 @@
...
+
+**Step 10.** Enter or select the name of Resource Group and Application Insights Resource created earlier. Then press the "OK" button to add the telemetry in the project.
+
+![](<media/prereq-step8.png>)
+
+**Step 11.** Back in Visual Studio, open appsettings.json in PartsUnlimitedWebsite project, note "ApplicationInsights" is added.
+
+![](<media/prereq-step11.png>)
+
+**Step 12.** Open Startup.cs in PartsUnlimitedWebsite project. Add the following code in Startup method.
```
Comment (steven2000nz) [inline docs/HOL-Usage_Analysis_With_Application_Insights/README.md]: Sorry, I updated this code and image, such as 
.SetBasePath(env.ContentRootPath)
.AddJsonFile("config.json")
.AddJsonFile("appsettings.json", optional: true, reloadOnChange: true)

File: docs/HOL-Usage_Analysis_With_Application_Insights/README.md [hunk 2]
```diff
@@ -0,0 +1,377 @@
...
+ window.appInsights.stopTrackEvent(recordToDelete);
+...
+~~~~
+
+### Task 3: Create a custom telemetry event on the server side
+**Step 1.**
+Firstly we need an Application Insights telemetry provider. This will essentially wrap up any Application Insights specific calls inside an common interface. This means we can swap out specific tracing implementations either before or at runtime.
+
+![](<media/add-new-telemetry-class.png>)
+
```
Comment (steven2000nz) [inline docs/HOL-Usage_Analysis_With_Application_Insights/README.md]: The setting in Task1 -> Step 11 will overwrite this one.

File: docs/HOL-Usage_Analysis_With_Application_Insights/README.md (hunk 1 above)
Comment (jonocairns) [inline docs/HOL-Usage_Analysis_With_Application_Insights/README.md]: Fixed

File: docs/HOL-Usage_Analysis_With_Application_Insights/README.md (hunk 2 above)
Comment (jonocairns) [inline docs/HOL-Usage_Analysis_With_Application_Insights/README.md]: It's a different step so leaving this in

**Pull request merged**